design/cmd/other.toml

```diff
@@ -4,11 +4,19 @@ text = '''
 formats. This will allow a user to display their artifacts in a pretty
 format on services like github.
 
-Syntax: `art export TYPE`
+Syntax: `art export [-d|--output-directory PATH] TYPE`
 
 Supported types **shall** be:
 - html: this is the MUST HAVE export type
 - markdown: this is a possible future type
+
+`art export` **will** provide a flag `-d|--output-directory` that allows
+the user to specify a different output directory for the exported files.
+
+`art export` **will** support both the short form `-d` as the long form
```

Comment:
I think these four lines can be removed. Also, these are all "shall" requirements (they can be implemented)